### DISCUSSION TITLE
I present

### DISCUSSION THREAD

**Discussion by uiee** [ScratchAddons/ScratchAddons#3935] [Show and tell]
Scratch 2.0 style
![Screenshot 2021-12-14 3 42 50 PM](https://user-images.githubusercontent.com/93910541/146461370-20fd5044-2292-4a68-9014-bd5b5888ec87.png)

**Comment by Secret-chest** [comment]
Just need to move green flag to the right

**Comment by retronbv** [comment]
Hmmm, maybe an add on for the old flag icons?

**Comment by kccuber-scratch** [comment]
Use https://github.com/ScratchAddons/ScratchAddons/discussions/3871#discussioncomment-1794153 if you want the font back too
